platform/api/aws/ec2.go

```diff
@@ -240,27 +252,243 @@ func (a *API) GetConsoleOutput(instanceID string) (string, error) {
 // If the security group does not exist, it's created.
 func (a *API) getSecurityGroupID(name string) (string, error) {
 	sgIds, err := a.ec2.DescribeSecurityGroups(&ec2.DescribeSecurityGroupsInput{
-		GroupNames: []*string{&name},
+		Filters: []*ec2.Filter{
+			{
+				Name:   aws.String("group-name"),
+				Values: []*string{&name},
+			},
+		},
 	})
-	if isSecurityGroupNotExist(err) {
+
+	if len(sgIds.SecurityGroups) == 0 {
 		return a.createSecurityGroup(name)
```

Comment:
I realize this wasn't change in this PR but it may be worth updating in the future if there is a possibility that creating a security group could error.

Reply:
`createSecurityGroup` also returns `string, error` so if this function were to fail it would bubble out properly and should be detected in whatever function references this one.